factor/lists.py

```diff
@@ -11,40 +17,78 @@ def summation(numbers: list[int]) -> int:
 # ex: [11, 13, -1, 0, 9] -> 2
 def find_negative(numbers: list[int]) -> int:
     # TODO implement
+    # know that we have to put an empety string if you do the while loop
+    for index in range(len(numbers)) :
```

Comment:
watch style, colon characters should not have space preceding ie: for index in range(len(numbers)):